modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java

```diff
@@ -1563,6 +1613,41 @@ else if (CMD_SKIP_ZEROS.equals(nextArg))
 
                 break;
 
+            case DISTRIBUTION:
+                String nodeIdStr = nextArg("Node id expected or null");
+                if (!"null".equals(nodeIdStr))
+                    cacheArgs.nodeId(UUID.fromString(nodeIdStr));
+
+                while (hasNextCacheArg()) {
+                    String nextArg = nextArg("");
+
+                    if (CMD_USER_ATTRIBUTES.equals(nextArg)){
+                        nextArg = nextArg("User attributes are expected to be separated by commas");
+
+                        Set<String> userAttributes = new HashSet();
+
+                        for (String userAttribute:nextArg.split(","))
+                            userAttributes.add(userAttribute.trim());
+
+                        cacheArgs.setUserAttributes(userAttributes);
+
+                        if (hasNextCacheArg())
```

Comment:
Why not use ternary operator? as nextArg = hasNextCacheArg() ? nextArg("") : null;